src/lib/core/components/computations/RunComputeButton.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { FilledButton } from '@veupathdb/coreui';
+import { capitalize } from 'lodash';
+import { ComputationAppOverview } from '../../types/visualization';
+import { Tooltip } from '@veupathdb/components/lib/components/widgets/Tooltip';
+import { JobStatus } from './ComputeJobStatusHook';
+
+interface Props {
+  computationAppOverview: ComputationAppOverview;
+  status?: JobStatus;
+  createJob: () => void;
+}
+
+export function RunComputeButton(props: Props) {
+  const { computationAppOverview, status, createJob } = props;
+
+  return computationAppOverview.computeName ? (
+    <div
+      style={{
+        display: 'flex',
+        gap: '1em',
+        alignItems: 'center',
+      }}
+    >
+      <FilledButton
+        themeRole="primary"
+        text="Run computation"
+        textTransform="none"
+        size="small"
+        onPress={createJob}
+        disabled={status !== 'no-such-job'}
+      />
+      <div
+        style={{
+          display: 'inline-flex',
+          gap: '.5em',
+          fontWeight: 'bold',
+        }}
+      >
+        Status:{' '}
+        {status ? <StatusIcon status={status} showLabel /> : 'Loading...'}
```

Comment:
I thought I already commented on "no such job" being a bit offputting to users - maybe it was in Slack.  To me it sounds like an error has occurred.

Perhaps we can just not display any text at all?

`<StatusIcon status={status} showlabel={status !== 'no-such-job'}/>`